client/signup/steps/rewind-add-creds/index.jsx

```diff
@@ -0,0 +1,74 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import Card from 'components/card';
+import Button from 'components/button';
+
+class RewindAddCreds extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	goToCredsForm = () => {
+		SignupActions.submitSignupStep( {
+			processingMessage: this.props.translate( 'Preparing credentials form' ),
+			stepName: this.props.stepName,
+		} );
+
+		this.props.goToStep( 'rewind-form-creds' );
+	};
+
+	stepContent = () => {
+		const { translate } = this.props;
+
+		return (
+			<Card className="rewind-add-creds__card rewind-switch__card rewind-switch__content">
+				<h3 className="rewind-add-creds__title rewind-switch__heading">
+					{ translate( 'Add your credentials' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add credentials' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).